addons/rose/addon/components/rose/form/index.hbs

```diff
@@ -22,7 +22,6 @@
         'rose/form/checkbox/group' disabled=this.disabled
       )
       input=(component 'rose/form/input' disabled=this.disabled)
-      select=(component 'rose/form/select' disabled=this.disabled)
```

Comment:
Shouldn't this be replaced with the HDS component?

Reply:
It may not need to be.  The only reason to do this was to get the disabled state piped through.  However, this is easy to do without yielding.  As long as the form itself yields its state, we can do this:

```
<Hds::Form::Select::Field disabled={{form.disabled}} as |F|>
```